From 7037cadd0c64c52269c45047b1543b1e53663bc7 Mon Sep 17 00:00:00 2001
From: Ludovic Desroches <ludovic.desroches@atmel.com>
Date: Mon, 9 Jul 2012 08:51:50 +0200
Subject: [PATCH 9/9] mmc: atmel-mci: increase dma threshold

There are some issues with some SD cards when dma is used. DMA transfer
during cmd6 seems to hang for an unknown reason. Since using PIO prevents from
this issue, the dma threshold has been increased to not use dma for this
command.

Signed-off-by: Ludovic Desroches <ludovic.desroches@atmel.com>
---
 drivers/mmc/host/atmel-mci.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/atmel-mci.c b/drivers/mmc/host/atmel-mci.c
index a6f885c..d2a8aec 100644
--- a/drivers/mmc/host/atmel-mci.c
+++ b/drivers/mmc/host/atmel-mci.c
@@ -45,7 +45,7 @@
 #include "atmel-mci-regs.h"
 
 #define ATMCI_DATA_ERROR_FLAGS	(ATMCI_DCRCE | ATMCI_DTOE | ATMCI_OVRE | ATMCI_UNRE)
-#define ATMCI_DMA_THRESHOLD	16
+#define ATMCI_DMA_THRESHOLD	65
 
 enum {
 	EVENT_CMD_RDY = 0,
-- 
1.7.10.4

